src/pages/data-visualization/advanced-charts/index.mdx

```diff
@@ -17,97 +17,404 @@ Advanced charts are a powerful way to display complex data sets. While less intu
 
 <AnchorLinks>
 
-<AnchorLink>Alluvial diagram</AnchorLink>
-<AnchorLink>Heatmap</AnchorLink>
-<AnchorLink>Tree diagram</AnchorLink>
-<AnchorLink>Geography with overlays</AnchorLink>
+<AnchorLink>Alluvial diagrams</AnchorLink>
```

Comment:
I think this needs to be changed to Alluvial diagram (or the heading needs to be changed (doesn't have an s), right now clicking on the anchor link doesn't bring me to this section since the anchors do not match.

Reply:
updated!